editor/editor_about.cpp

```diff
@@ -141,20 +141,20 @@ ScrollContainer *EditorAbout::_populate_list(const String &p_name, const List<St
 					const String identifier = name.get_slice("<", 0);
 					const String website = name.get_slice_count("<") == 1 ? "" : name.get_slice("<", 1).trim_suffix(">");
 
-					const int name_item_id = il->add_item(identifier, nullptr, false);
-					il->set_item_tooltip_enabled(name_item_id, false);
+					il->add_item(identifier, nullptr, false);
+					il->set_item_tooltip_enabled(-1, !website.is_empty());
 
 					if (!website.is_empty()) {
-						il->set_item_selectable(name_item_id, true);
-						il->set_item_metadata(name_item_id, website);
-						il->set_item_tooltip(name_item_id, website + "\n\n" + TTR("Double-click to open in browser."));
-						il->set_item_tooltip_enabled(name_item_id, true);
-					}
-
-					if (!*names_ptr && name.contains(" anonymous ")) {
-						il->set_item_disabled(name_item_id, true);
+						il->set_item_selectable(-1, true);
+						il->set_item_metadata(-1, website);
+						il->set_item_tooltip(-1, website + "\n\n" + TTR("Double-click to open in browser."));
 					}
 				}
+				if (p_anonymous_counts && p_anonymous_counts[i] > 0) {
+					il->add_item(vformat(TTRN("And %d anonymous donor", "And %d anonymous donors", p_anonymous_counts[i]), p_anonymous_counts[i]), nullptr, false);
+					il->set_item_tooltip_enabled(-1, false);
```

Comment:
Doesn't look like there's any need to disable the tooltip here.

Reply:
This is the same behavior as before.

The entry doesn't show tooltip since it doesn't have a website link.